packages/hydrogen-react/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @shopify/hydrogen-react
 
+## 2024.0.0
+
+### Major Changes
+
+- Release 2023-04 by [@lordofthecactus](https://github.com/lordofthecactus)
```

Comment:
Do we have release notes / changes? Including info on breaking changes in our library and any significant changes in the SFAPI?